### DISCUSSION TITLE
Code style not working for title

### DISCUSSION THREAD

**Discussion by Mysterious-Dev** [community/community#43421] [Mobile]
### Select Topic Area

Bug

### Body

Code style (`like this`) for title not working on mobile version.

System : `Android`
Version : `Android 13`
Phone model : `Samsung Galaxy A52s 5G`
Github Version : `1.96.0`

## Github Desktop

![Github desktop title](https://user-images.githubusercontent.com/40738104/210882036-0014585b-2e50-4410-9358-277a26a11c8e.png)
![Github desktop view](https://user-images.githubusercontent.com/40738104/210882040-ffca9c8c-0684-48b5-bd26-5cceac980af2.png)

## Github Mobile

![Github mobile title](https://user-images.githubusercontent.com/40738104/210882041-b308e291-f46b-4731-84e8-283ba715c222.jpg)
![Github mobile view](https://user-images.githubusercontent.com/40738104/210882030-dc96d994-fd1c-4bc3-a82f-15402e5b603b.jpg)

**Comment by jfullstackdev** [comment]
yup, there were several reported bugs for the mobile,

if you still want to have that as a workaround, use the Chrome in your mobile to have almost same experience as GitHub Web

**Reply by Mysterious-Dev** [reply]
It's not really disturbing in itself. I just report the bug so that it will be fixed one day.
